{
  "Faults": [
    {
      "latitude": 51.18750923556919,
      "type": "HV",
      "reference": "KP4868",
      "etaUpdate": true,
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "698579a1-01ec-42c2-a0e6-3a14340e86ef",
      "minLongitude": -1.869,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8062557003002189,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-24T15:53:00.000+0000",
      "estimatedRestoration": "2023-02-24T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KP4868'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KP4868",
      "maxLongitude": -1.8397,
      "minLatitude": 51.1844,
      "name": "SP3 & SP4 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "SP3 4DX",
        "SP4 7DA",
        "SP4 7DD",
        "SP4 8BN",
        "SP4 8BW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.869,
              51.2062
            ],
            [
              -1.8397,
              51.2062
            ],
            [
              -1.8397,
              51.1844
            ],
            [
              -1.869,
              51.1844
            ],
            [
              -1.869,
              51.2062
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2062,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-24T13:40:00.000+0000",
      "latitude": 51.459177099984935,
      "estimatedRestoration": "2023-02-24T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KP4820'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KP4820",
      "reference": "KP4820",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-02-24T17:10:00.000+0000",
      "affectedCustomerCount": 2,
      "name": "TW19 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "TW19 7NS"
      ],
      "location": {
        "coordinates": [
          -0.49313573854978604,
          51.459177099984935
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2023-02-24T17:30:00.000+0000",
      "UUID": "39d050ac-2794-403f-8373-de3eb988682c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.49313573854978604,
      "resolved": false
    },
    {
      "latitude": 51.38252577531193,
      "type": "LV",
      "reference": "KP4637",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-02-24T15:25:00.000+0000",
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d3aec0b9-5016-41a5-86b7-1d1b6a756ca6",
      "minLongitude": -2.1437,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.141789992988429,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-24T12:32:00.000+0000",
      "estimatedRestoration": "2023-02-24T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KP4637'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KP4637",
      "maxLongitude": -2.1374,
      "minLatitude": 51.381,
      "name": "SN12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN12 8AE",
        "SN12 8AL",
        "SN12 8AN",
        "SN12 8AS",
        "SN12 8AT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1437,
              51.384
            ],
            [
              -2.1374,
              51.384
            ],
            [
              -2.1374,
              51.381
            ],
            [
              -2.1437,
              51.381
            ],
            [
              -2.1437,
              51.384
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.384,
      "etrUpdate": true
    },
    {
      "latitude": 51.5824554059781,
      "type": "LV",
      "reference": "KP4905",
      "etaUpdate": true,
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ddd5e0d9-93c5-46ee-a6e9-8ad4717d6a4a",
      "minLongitude": -1.7758,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.774275410244965,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-24T16:24:00.000+0000",
      "estimatedRestoration": "2023-02-24T21:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KP4905'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KP4905",
      "maxLongitude": -1.7709,
      "minLatitude": 51.5809,
      "name": "SN2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN2 7BD",
        "SN2 7BE",
        "SN2 7BG",
        "SN2 7HH",
        "SN2 7HQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7758,
              51.5854
            ],
            [
              -1.7709,
              51.5854
            ],
            [
              -1.7709,
              51.5809
            ],
            [
              -1.7758,
              51.5809
            ],
            [
              -1.7758,
              51.5854
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-24T17:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5854,
      "etrUpdate": true
    },
    {
      "latitude": 57.46553200436568,
      "type": "HV",
      "reference": "KP4991",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "73363c5c-7c5d-4d07-99e2-739d57d69873",
      "minLongitude": -7.4146,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -7.36261540856213,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-24T18:32:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KP4991'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KP4991",
      "maxLongitude": -7.2634,
      "minLatitude": 57.4198,
      "name": "HS7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HS7 5LA",
        "HS7 5LT",
        "HS7 5LY",
        "HS7 5QA",
        "HS7 5QB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -7.4146,
              57.5043
            ],
            [
              -7.2634,
              57.5043
            ],
            [
              -7.2634,
              57.4198
            ],
            [
              -7.4146,
              57.4198
            ],
            [
              -7.4146,
              57.5043
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "209289ed-0ad3-4599-8ad6-bb8a597849c1"
      ],
      "maxLatitude": 57.5043,
      "etrUpdate": true
    }
  ]
}
